app/assets/stylesheets/card.scss

```diff
@@ -0,0 +1,18 @@
+$card-box-shadow: 3px 5px 8px rgba(0,0,0,.4);
```

Comment:
`.4` should be written with a leading zero as `0.4`<br>Commas in function arguments should be followed by a single space